src/about.njk

```diff
@@ -142,7 +142,7 @@
         <p class="text-muted">Birmingham Leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://www.linkedin.com/in/jacob-watson-007/" target="_blank" class="shadow-sm">
+            <a href="https://www.linkedin.com/in/jacob-watson-007/" target="_blank" class="shadow-sm" aria-label="Navigation link to Jacob Watson LinkedIn Page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Jacob Watson's LinkedIn Page"